src/packages/client/src/__tests__/integration/happy/exhaustive-schema/__snapshots__/test.ts.snap

```diff
@@ -1624,48 +1761,94 @@ export namespace Prisma {
     findOne<T extends FindUniquePostArgs>(
       args: Subset<T, FindUniquePostArgs>
     ): CheckSelect<T, Prisma__PostClient<Post | null>, Prisma__PostClient<PostGetPayload<T> | null>>
+
     /**
-   * Count the number of Posts.
-   * @param {FindManyPostArgs} args - Arguments to filter Posts to count.
-   * @example
-   * // Count the number of Posts
-   * const count = await prisma.post.count({
-   *   where: {
-   *     // ... the filter for the Posts we want to count
-   *   }
-   * })
-  **/
+     * Count the number of Posts.
+     * @param {FindManyPostArgs} args - Arguments to filter Posts to count.
+     * @example
+     * // Count the number of Posts
+     * const count = await prisma.post.count({
+     *   where: {
+     *     // ... the filter for the Posts we want to count
+     *   }
+     * })
+    **/
     count(args?: Omit<FindManyPostArgs, 'select' | 'include'>): Promise<number>
 
-    
     /**
-   * Group By
-  **/
-    groupBy<T extends GroupByPostArgs>(args: Subset<T, GroupByPostArgs>): Promise<GetPostGroupByPayload<T>>
+     * Allows you to perform aggregations operations on a Post.
+     * @param {AggregatePostArgs} args - Select which aggregations you would like to apply and on what fields.
+     * @example
+     * // Ordered by age ascending
+     * // Where email contains prisma.io
+     * // Limited to the 10 users
+     * const aggregations = await prisma.user.aggregate({
+     *   avg: {
+     *     age: true,
+     *   },
+     *   where: {
+     *     email: {
+     *       contains: "prisma.io",
+     *     },
+     *   },
+     *   orderBy: {
+     *     age: "asc",
+     *   },
+     *   take: 10,
+     * })
+    **/
+    aggregate<T extends AggregatePostArgs>(args: Subset<T, AggregatePostArgs>): Promise<GetPostAggregateType<T>>
 
     /**
-   * Allows you to perform aggregations operations on a Post.
-   * @param {AggregatePostArgs} args - Select which aggregations you would like to apply and on what fields.
-   * @example
-   * // Ordered by age ascending
-   * // Where email contains prisma.io
-   * // Limited to the 10 users
-   * const aggregations = await prisma.user.aggregate({
-   *   avg: {
-   *     age: true,
-   *   },
-   *   where: {
-   *     email: {
-   *       contains: "prisma.io",
-   *     },
-   *   },
-   *   orderBy: {
-   *     age: "asc",
-   *   },
-   *   take: 10,
-   * })
-  **/
-    aggregate<T extends AggregatePostArgs>(args: Subset<T, AggregatePostArgs>): Promise<GetPostAggregateType<T>>
+     * Group By
+    **/
+    groupBy<
```

Comment:
Could use a description and example probably? Like count & aggregate for example

Reply:
Good point!